crates/egui/src/memory.rs

```diff
@@ -191,6 +191,10 @@ pub struct Options {
     /// Controls the tessellator.
     pub tessellation_options: epaint::TessellationOptions,
 
+    /// If `true`, This will call `egui::Context::request_repaint()` at the end of each frame
+    /// If `false` (default), egui is only updated if are input events (like mouse movements) or there are some animations in the GUI.
+    pub continuous_mode: bool,
```

Comment:
Maybe we should just move `enum RunMode {` into `egui` from `crates/egui_demo_app/src/backend_panel.rs`. At least we should copy its documentation.